pkg/cmd/roachtest/jobs.go

```diff
@@ -138,16 +145,17 @@ func jobSurvivesNodeShutdown(
 	})
 
 	// Before calling `m.Wait()`, do some cleanup.
-	if err := m.g.Wait(); err != nil {
-		t.Fatal(err)
-	}
+	//if err := m.g.Wait(); err != nil {
```

Comment:
hmmm this diff seems to be unique to this backport. Can you explain why this changed?

Reply:
sorry, i should mark this pr as a draft. While resolving the merge conflict, i left a few things commented because i didn't understand the full implications of my changes to 21.1.  I need to test the change, but i haven't been able to run the roachtest -- https://cockroachlabs.slack.com/archives/CJ0H8Q97C/p1638897754468000

Reply:
@adityamaru just pushed a revision which removed this change.